util/markup.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+from dominate.tags import a
```

Comment:
Maybe. I can‘t say right now.

I just didn‘t feel like building tag(s) myself and re-implementing the wheel. And since this lib seems kind of maintained and has no other dependencies I felt okay adding it.